src/common/constants.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	APP_VERSION = "v0.4rc"
+	APP_VERSION = "v0.5rc"
```

Comment:
v0.4 hasn't been release yet, this is not time for v0.5

Reply:
it was only a test to see if we get our local change inside docker machine, i revert